tests/hub/test_download_dataset_file.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+
+import os
+import tempfile
+import time
+import unittest
+
+from modelscope.hub.dataset_download import (dataset_file_download,
+                                             dataset_snapshot_download)
+
+
+class DownloadDatasetTest(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_dataset_file_download(self):
+        dataset_id = 'citest/test_dataset_download'
```

Comment:
also test wildcard characters

Reply:
It has been added in dataset_snapshot_download,  dataset_file_download only supports exact path.